src/object/game/arena.lua

```diff
@@ -6,6 +6,7 @@ function Arena:init(x, y, shape)
     self:setOrigin(0.5, 0.5)
 
     self.color = {0, 0.75, 0}
+    self.bgColor = {0, 0, 0}
```

Comment:
The naming scheme for variables is `snake_case`, so `self.bg_color` is better here. Also, all objects have `setColor` and `getColor`, so I'd recommend implementing `setBackgroundColor` and `getBackgroundColor` for the arena.